src/routes/blog/spyder-6-daniel/+page.md

```diff
@@ -0,0 +1,103 @@
+---
+title: "Spyder 6 under the hood: Editor migration, remote dev QA, test overhaul and more!"
+author:
+  - camgerlach
+  - dalthviz
+  - ccordoba12
+tags: Feature, Spyder6
+category: Releases
+pub_date: 2024-12-13
+summary: "Beyond the headline features, there's a lot more new and improved under the hood in Spyder 6. Daniel Althviz, Spyder's release manager and co-maintainer, was at the forefront of much of it, and we're here to share the highlights with all of you and what he plans to work on next!"
+---
+
+[Daniel Althviz](https://github.com/dalthviz) is one of our longest-tenured Spyder developers, having started working on the project near the end of 2016.
+Now eight years later, he's the project's co-maintainer and its our release manager, responsible for the many steps required to release Spyder across all our various platforms.
+On top of that, he is also the lead maintainer of two of the most important libraries on which Spyder, and a lot of other scientific software, depends: the [QtPy](https://github.com/spyder-ide/qtpy) GUI abstraction layer and the [QtAwesome](https://github.com/spyder-ide/qtawesome) icon font framework.
+
+In 2023, Spyder received a [Chan Zuckerberg Initiative](https://chanzuckerberg.com/) Essential Open Source Software for Science Cycle 5 grant, [number 384](https://chanzuckerberg.com/eoss/proposals/enhancing-spyder-ide-remote-support-for-scientific-research-in-python/) to support a new remote development architecture and features in Spyder and related development.
```

Comment:
I think here the parentheses from @ccordoba12 suggestion are missing?
```suggestion
In 2023, Spyder received a [Chan Zuckerberg Initiative](https://chanzuckerberg.com/) Essential Open Source Software for Science Cycle 5 grant ([number 384](https://chanzuckerberg.com/eoss/proposals/enhancing-spyder-ide-remote-support-for-scientific-research-in-python/)) to support a new remote development architecture and features in Spyder and related development.
```

Reply:
Also, using the actual proposal name (`Enhancing Spyder IDE Remote Support for Scientific Research in Python`) than its number could make more sense?

Reply:
Hey, we actually went through all the suggestions in our meeting together today and adjusted the text per my applied suggestion, since the grant number is important to include per standard attribution convention and the link refers to our specific grant, that number, as opposed to cycle 5 in general. Thanks for making sure!